alda-mode.el

```diff
@@ -277,6 +277,29 @@ Because alda runs in the background, the only way to do this is with alda restar
   ;; Set alda highlighting
   (setq font-lock-defaults '(alda-highlights)))
 
+(defun alda-play-block ()
+  (interactive)
+  (let* ((p (point)))
+	(mark-paragraph)
+	(alda-play-region (region-beginning) (region-end))
+	(goto-char p)))
+
+(defun alda-play-line ()
+  (interactive)
+  (alda-play-region (line-beginning-position) (line-end-position)))
+
+(defun alda-play-buffer ()
+  (interactive)
+  (let* ((p (point)))
+    (mark-whole-buffer)
+    (alda-play-region (region-beginning) (region-end))
+    (goto-char p)))
+
+(progn
```

Comment:
Could you add a variable to opt out of these keybindings if someone wanted to (just in case someone had already bound these in their personal maps).